File: Simulator-VB/Models/Box.vb
Namespace Models

    <Serializable()>
    Public Class Box

        Sub New()
        End Sub

        Sub New(value As Double, amount As Integer)
            Me.New()
            Me.Value = value
            Me.Amount = amount
        End Sub

        Public Property Value As Double
        Public Property Amount As Integer

        Public ReadOnly Property SubTotal As Double
            Get
                Return Value * Amount
            End Get
        End Property

    End Class

End Namespace

File: Simulator-CSharp/Models/Box.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ATM.Models
{
    public class Box
    {

        public Box()
        {
        }

        public Box(double value, int amount)
        {
            Value = value;
            Amount = amount;
        }

        public double Value { get; set; }

        public int Amount { get; set; }

        public double SubTotal
        {
            get
            {
                return Value * Amount;
            }
            private set { }
        }

    }
}
